### PR TITLE
added zlh's blog

### PR DESCRIPTION
**Pull request by mathsyouth**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.  
    For example:  

    ``` md
    ---
    title: "Your Great Blog Title"
    author: your_github_id
    image: /assets/2020/08-your-blog-slug/teaser.jpg
    ---

    YOUR_FIRST_SENTENCE_SHOULD_BE_A_SUMMARY_FOR_YOUR_POST
    ```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2020` directory.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

To read more about the contribute guideline, visit: <https://github.com/wechaty/wechaty.js.org#contribute-guideline>

### CODE REVIEW

File: jekyll/_posts/2020-08-29-wx-group-assistant-bot.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: "微信营销群机器人客服"
+author: zlh
+categories: project
+tags:
+  - wechaty
+  - nodejs
+  - wechaty-puppet-padplus
```
Comment (lijiarui) [inline jekyll/_posts/2020-08-29-wx-group-assistant-bot.md:9]: Please add an image for this file.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

File: jekyll/_posts/2020-08-29-wx-group-assistant-bot.md (hunk 1 above)
Comment (mathsyouth) [reply jekyll/_posts/2020-08-29-wx-group-assistant-bot.md:9]: Sorry for my mistake. I have added an image for this file. Thanks for your review.

**Review by lijiarui — APPROVED** [review]

**Comment by lijiarui** [comment]
Congratulations! 🎉 🎉 🎉

You become a wechaty contributor successfully! 👏 👏 👏

Please:

- [ ] Update your profile here: [Contributor List](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0), then other contributors will know you better!
- [ ]  Join wechaty org by clicking https://github.com/wechaty
- [ ]  Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room

**Pull request merged**